src/sentry/api/serializers/models/project.py

```diff
@@ -390,6 +423,23 @@ def get_session_stats(self, project_ids):
 
         return project_health_data_dict
 
+    def get_options(self, projects):
+        # no options specified
+        option_list = []
+
+        # must be a safe key
+        if self.expand_context.get("options"):
+            option_list = self.expand_context.get("options")
+            option_list = [option for option in option_list if option in OPTION_KEYS]
+
+        queryset = ProjectOption.objects.filter(project__in=projects, key__in=option_list)
+
+        options_by_project = defaultdict(dict)
+        for option in queryset.iterator():
```

Comment:
```suggestion
        for option in queryset:
```

You shouldn't need to call `.iterator()` explicitly.